project 3/CODE/index.html

```diff
@@ -0,0 +1,140 @@
+
+            Don't get me wrong I love Graphic Design. It just gets to be a lot sometimes. I need a good cry from time to
+            time you know? I dont know maybe you do or don't. Live your life. </p>
+    </div>
+    <div id="hour9am">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour9am"><img src="images/9am.jpg" alt=''></a>
+        <h2>9:00 AM</h2>
+        <p>I never stop working. Which might be why I am always tired but none the less I tend to work on multiple
+            different things in a short amount of time. Not that I am saying that I am a multi-tasker. but from what I
+            have seen of my own proccess is that I tend to work better in a constant change in work. So for me I find
+            that it is easiest to work on something for an hour and then take a break by working on another project for
+            an hour and just rotating through all of the work I have on my plate. At this moment I am working on the
+            final for my writing class "Playing With Words", for which I am working on a book of love poetry about my
+            past my prestn and future.
+
+            I am so ready for summer after all of the work I have made this semester. I am read to delve into my own
+            work with no distractions like classes and things of that nature to get in my way.</p>
+    </div>
+    <div id="hour10am">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour10am"><img src="images/10am.jpg" alt=''></a>
+        <h2>10:00 AM</h2>
+        <p>Here I am just looking for ideas on how to sturcture some of my poetry for my zine. Which leads me into two
+            different finals in two different classes. But both are gonna take a lot of work to acomplish in the time
+            frame of just one month.</p>
+    </div>
+    <div id="hour11am">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour11am"><img src="images/11am.jpg" alt=''></a>
+        <h2>11:00 AM</h2>
+        <p>Now I am back at it tking notes in Gen Type. Someone told me this is the most difficult class that the
+            Graphic Design Department has to offer? I have no idea. Someone wanna answer to the contrary? I mean either
+            way I really enjoy this class even though it does hand me my behind mentally and physically. Luckily
+            Angelique is super chill so it this class is a good way to end the day.</p>
+    </div>
+    <div id="hour12pm">
+        <a href="#" class="button-close">&times;</a>
+        <a href="#hour12pm"><img src="images/12pm.jpg" alt=''></a>
+        <h2>12:00 PM</h2>
+        <p>Now I need sustinence. I have this really bad habit of not eating breakfast. What is that about? I mean I
+            chalk it up to just being an artist. I need to fix that I really need to work on what my morning routine
+            looks like. Right now it is in no way shape or form healthy or sustainable. HELP!!</p>
```

Comment:
I don't think college-age Angelique even knew morning routines were a thing, so you're probably ahead of me there. Adult-me is very pro-breakfast, though.